tower/CHANGELOG.md

```diff
@@ -6,17 +6,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
+None.
+
+# 0.4.5 (February 10, 2021)
+
+### Added
+
 - **util**: Add `ServiceExt::map_future`. ([#542])
-- **util**: Make combinators implement `Debug` in more cases
-- `tracing` is now only pulled in for the features that need it.
 - **util**: Add `option_layer` to convert an `Option<Layer>` into a `Layer`. ([#555])
 - **builder**: Add `ServiceBuilder::option_layer` to optionally add a layer. ([#555])
-- **steer**: `Steer` now implements `Clone
 - **make**: Added `Shared` which lets you implement `MakeService` by cloning a
```

Comment:
nit: the tense here is inconsistent with the other entries ("Added ..." rather than "Add  ..."). We should _probably_ try to keep that consistent across releases, but let's at least keep it consistent here?

(it's entirely possible that i'm the only person in the world who cares about this, sorry!)

Reply:
Its all about the little things 😊